src/app/backend/scaling/scale.go

```diff
@@ -92,7 +92,7 @@ func getScaleGetter(cfg *rest.Config) (scale.ScalesGetter, error) {
 		return nil, err
 	}
 
-	cfg.GroupVersion = &apps.SchemeGroupVersion
+	cfg.GroupVersion = &appsv1beta2.SchemeGroupVersion
```

Comment:
Why?

Reply:
I make a mistake, please close this pr, I will add a new one.